### PR TITLE
Paraphrase complicated negated sentence

### PR DESCRIPTION
**Pull request by phannebohm**

### CODE REVIEW

**Comment by phannebohm** [comment]
Conceptually, should this also extend to chains of more than two operator records like `C->D->E->C`?
(`->` represents a constructor, so the described scenario is `C->D->C`)

File: chapters/overloaded.tex [hunk 1]
```diff
@@ -139,15 +139,15 @@ \section{Overloaded Constructors}\label{overloaded-constructors}
...
+  For an operator record class there shall not exist any potential call
   that lead to multiple matches in (1) above.
   \begin{nonnormative}
   How to verify this is not specified.
   \end{nonnormative}
 \item
   For a pair of operator record classes \lstinline!C! and \lstinline!D! and components \lstinline!c! and \lstinline!d!
-  of these classes both of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
-  shall not both be legal.
+  of these classes at most one of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
```
Comment (henrikt-ma) [inline chapters/overloaded.tex:149]: This is one of the many places where I suppose @GallLeo's grammar check would suggest to use a comma after _of these classes_.
```suggestion
  of these classes, at most one of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
```

**Review by henrikt-ma — APPROVED** [review]
Looks good to me.  Just leaving a minor comment.

**Comment by henrikt-ma** [comment]
> Conceptually, should this also extend to chains of more than two operator records like `C->D->E->C`?
> (`->` represents a constructor, so the described scenario is `C->D->C`)

This looks like a good question to ask in a new issue.

**Comment by HansOlsson** [comment]
Since you are a new contributor we also want a signed Contributor's License Agreement, CLA (or being part of an organization having signed the CLA on your behalf). @GallLeo

**Comment by HansOlsson** [comment]
> Conceptually, should this also extend to chains of more than two operator records like `C->D->E->C`?
> (`->` represents a constructor, so the described scenario is `C->D->C`)

I don't see how that could occur. Constructors do not allow implicit calls of multiple constructors, and for the operators they are at most binary so we only have two arguments.

**Review by HansOlsson — CHANGES_REQUESTED** [review]
Change looks good, but we need the CLA issue clarified.

**Comment by phannebohm** [comment]
> Change looks good, but we need the CLA issue clarified.

No problem, except how do I sign? :sweat_smile: 
I'm with the university of applied sciences in Bielefeld, they may have signed it.

**Comment by HansOlsson** [comment]
> > Change looks good, but we need the CLA issue clarified.
> 
> No problem, except how do I sign? 😅
> I'm with the university of applied sciences in Bielefeld, they may have signed it.

I understand you have to sign https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1/view and send it in some way
@GallLeo can you handle this?

**Comment by GallLeo** [comment]
I checked and FH Bielefeld has not signed he MA CLA, so far. 

Please hand in a signed scan to backoffice@modelica.org and on paper to 
MA Backoffice 
c/o LTX Simulation GmbH
Wohlfartstr. 21b
80939 München 

In schedule B, please list the employees, which should be covered by the CLA (best by name, email address and Github User Name).
After we do have this signed version, the "point of contact" can simply add or remove employees by sending informal e-mail (this will update schedule B).

**Review by eshmoylova — COMMENTED** [review]
Should probably also update the sentence on lines 164-166:
>Without mixed operations expressions such as
c+d are only ambiguous if both conversion from C to D and back from D to
C are both available...

"both" used twice

File: chapters/overloaded.tex [hunk 2]
```diff
@@ -139,15 +139,15 @@ \section{Overloaded Constructors}\label{overloaded-constructors}
...
+  For an operator record class there shall not exist any potential call
   that lead to multiple matches in (1) above.
   \begin{nonnormative}
   How to verify this is not specified.
   \end{nonnormative}
 \item
   For a pair of operator record classes \lstinline!C! and \lstinline!D! and components \lstinline!c! and \lstinline!d!
-  of these classes both of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
-  shall not both be legal.
+  of these classes, at most one of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
```
Comment (eshmoylova) [inline chapters/overloaded.tex:149]: I think "respectively" might be useful here:
```suggestion
  of these classes, respectively, at most one of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
```

Comment (phannebohm) [reply chapters/overloaded.tex:149]: True. And thanks for reminding me of this issue! :smiley:

**Comment by HansOlsson** [comment]
> I checked and FH Bielefeld has not signed the MA CLA, so far.
> 
> Please hand in a signed scan to [backoffice@modelica.org](mailto:backoffice@modelica.org) and on paper to
> MA Backoffice
> c/o LTX Simulation GmbH
> Wohlfartstr. 21b
> 80939 München
> 
> In schedule B, please list the employees, which should be covered by the CLA (best by name, email address and Github User Name).
> After we do have this signed version, the "point of contact" can simply add or remove employees by sending informal e-mail (this will update schedule B).

Any update w.r.t. CLA?
@GallLeo @phannebohm

**Comment by HansOlsson** [comment]
Still no CLA from Bielefeld as far I know. Reminder sent July 14th.

**Comment by HansOlsson** [comment]
Without CLA we cannot accept it.

**Comment by phannebohm** [comment]
The HSBI (former FH Bielefeld) managed to sign the CLA finally. It should have arrived at the Backoffice or be there shortly. Can you reopen this PR or should I open a new one?

**Comment by HansOlsson** [comment]
Have finally gotten CLA.

**Review by HansOlsson — APPROVED** [review]

**Pull request merged**